src/dmd/dinterpret.d

```diff
@@ -38,6 +38,7 @@ import dmd.identifier;
 import dmd.init;
 import dmd.initsem;
 import dmd.mtype;
+import dmd.printast;
```

Comment:
```suggestion
debug import dmd.printast;
```

Or delete ;)

Reply:
I keep needing to add it back in. It doesn't hurt to leave it in.

Reply:
With `debug` you don't have to remove it. It will be there just for your local debugging, but not compiled in the final release.

Reply:
One of the new developers on my team (who was primary a TypeScript developer before joining the team), told me that his favorite D feature is the ability to do:

```D
debug import std.stdio : writeln, writefln;

//...

debug
{
    // various writefln/writeln debug statements...
    writeln(...);
}
```

;)